app/app.js

```diff
@@ -135,6 +138,9 @@ app.post('/snap', [
   query('logo', `Must be one of the following values: ${Object.keys(logos).join(', ')}. If you would like to use your site's logo with Snap Service, please read how to add it at https://github.com/UN-OCHA/tools-snap-service#custom-logos`).optional().isIn(Object.keys(logos)),
   query('service', 'Must be an alphanumeric string identifier (hyphens, underscores are also allowed).').optional().matches(/^[A-Za-z0-9_\-]+$/),
   query('ua', '').optional(),
+  query('delay', 'Must be an integer between 0-10000 inclusive.').optional().isInt({ min: 0, max: 10000 }),
+  query('debug', 'Must be a Boolean').optional().isBoolean(),
+  query('block', 'Must be a comma-separated list of domains (alphanumeric, hyphens, dots, commas)').optional().matches(/^[A-Za-z0-9\.\,\-]+$/),
```

Comment:
The comma does not need to be escaped, domain names are not ascii.

Reply:
I just like escaping everything in my regexes because it hurts to think